compiler/passes/scopeResolve.cpp

```diff
@@ -1922,8 +1922,8 @@ static void resolveEnumeratedTypes() {
         if (EnumType* type = toEnumType(first->var->type)) {
           if (SymExpr* second = toSymExpr(call->get(2))) {
             const char* name;
-            bool found = false;
-
+            //bool found = false;
+            /*
```

Comment:
I probably could just remove the comments from that function entirely and document them here instead.  

I didn't use the same comment because it wasn't the same section of code. If I recall correctly there was a line of blank space in between them before.